Client/src/components/admin/Modal.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import { Modal, Button, Form, Row, Col } from 'react-bootstrap';
+class ModalDashBoard extends Component {
```

Comment:
React component should be have props type to define props from other classes ( to define schema and make constraint about data between component)
References: https://reactjs.org/docs/typechecking-with-proptypes.html